innovation.game.php

```diff
@@ -1682,6 +1761,12 @@ function notifyWithTwoPlayersInvolved($card, $transferInfo, $progressInfo) {
                 $message_for_others = clienttranslate('${player_name} transfers a ${<}${age}${>} from his hand to ${opponent_name}\'s score pile.');
                 break;
                 
+            case 'hand->achievements':
+                $message_for_player = clienttranslate('${You} transfer ${<}${age}${>} ${<<}${name}${>>} from your hand to ${opponent_name}\'s achievements.');
+                $message_for_opponent = clienttranslate('${player_name} transfers ${<}${age}${>} ${<<}${name}${>>} from his hand to ${your} achievements.');
```

Comment:
${player_name} transfers a ${<}${age}${>} from his hand to ${your} achievements.

Reply:
Even though it is going to their achievements, they aren't supposed to look at it.